client/state/help/ticket/test/reducer.js

```diff
@@ -1,72 +0,0 @@
-import {
-	HELP_TICKET_CONFIGURATION_REQUEST,
```

Comment:
Those actions can be deleted too.

Reply:
Done: https://github.com/Automattic/wp-calypso/pull/78433/commits/a4d654b99c671b7bd38f8524b28907ffc57e441f